### ISSUE TITLE
[1.16.5] Crash on startup: mcjty.theoneprobe.api.ITextStyle.getAlignment

### ISSUE THREAD

**Issue by CodeGordon** [McJty/TheOneProbe#499]
I've tried an existing world and a new world, both single-player, with and without `spawnNote` active.

[latest.log](https://github.com/McJtyMods/TheOneProbe/files/6841818/latest.log)

[crash-2021-07-19_10.32.58-server.txt](https://github.com/McJtyMods/TheOneProbe/files/6841797/crash-2021-07-19_10.32.58-server.txt)

**Comment by remplerus** [comment]
does it also happen, if you disable all mods? if so, it's a bug. If not, try to activate all mods one by one, start up. If it crashes, the bug happens on another mod that you activated last. just make an issue on their github, if you found the mod :)

**Comment by CodeGordon** [comment]
I fail to understand how a `NoSuchMethodError` would be caused by a mod conflict.

**Comment by McJty** [comment]
It's perfectly possible if another mod uses that method. Anyway TOP is working fine for others so we have to try this to see what's going on

**Comment by remplerus** [comment]
NoSuchMethodError means that a mod uses an old Method, which isn't part of the TOP code anymore

**Comment by remplerus** [comment]
i'm not 100% sure if it's this mod. but after disabling this mod, the game didn't crash 

Resynth - Growable Resources

after looking into their code, i found, that they overwrite(?) top's api

https://github.com/Resynth-Minecraft-Mod/Resynth-Mod/tree/1.16.5/src/main/java/mcjty/theoneprobe/api

running top and resynth alone doesn't crash, so i tried to see which mod might also tries to crash with those mods enabled but i couldn't reproduce without having almost all mods enabled

**Comment by CodeGordon** [comment]
Fair enough, I stand corrected. Their ITextStyle interface is empty, so if it's overwriting the TOP one, then that would explain the error. I figured things would be properly namespaced and referenced via APIs or exposing block/entity/etc data for other mods.

Their [mods.toml](https://github.com/Resynth-Minecraft-Mod/Resynth-Mod/blob/1.16.5/src/main/resources/META-INF/mods.toml) file doesn't list TOP as a dependency, so that might be doing it when Forge is loading multiple mods. I'll check that out and post the results.

**Comment by CodeGordon** [comment]
Well, that didn't work. Crash on load and the same error as before.
